test/config/FrontendAppConfigSpec.scala

```diff
@@ -8,15 +8,15 @@ import play.api.test.Helpers.GET
 class FrontendAppConfigSpec extends SpecBase {
 
   "FrontendAppConfig" - {
-    "feedbackUrl should include backUrl when request uri does not contain 'problem'" in {
+    "feedbackUrl should include backUrl when request uri does not contain 'there-is-a-problem'" in {
```

Comment:
need to update it in the `FrontendAppConfig` in the actual code too. So instead of
```
case uri if uri.contains("problem") => s"$feedbackLink"
```

update it to 
```
case uri if uri.contains("there-is-a-problem") => s"$feedbackLink"
```

Reply:
done